page/sass/footer.scss

```diff
@@ -8,11 +8,15 @@ html{
     position: relative;
 }
 .footer-container, .footer-bottom-container{
```

Comment:
fix this huge break please: 
![image](https://user-images.githubusercontent.com/1814312/29679633-6ffaf382-8902-11e7-8677-41723a65a59f.png)

Reply:
Now
<img width="244" alt="screen shot 2017-08-25 at 12 19 54 pm" src="https://user-images.githubusercontent.com/1808867/29710269-defd4c52-898f-11e7-9cd1-a0aeb07534a6.png">